optaplanner-examples/pom.xml

```diff
@@ -49,6 +49,9 @@
         <configuration>
           <ignoredUsedUndeclaredDependencies>
             <dependency>org.optaplanner:optaplanner-core-impl:jar</dependency>
+            <!-- optaplanner-persistence-jackson brings dependencies we do not want to enumerate. -->
+            <dependency>com.fasterxml.jackson.core:*</dependency>
+            <dependency>jakarta.xml.bind:*</dependency>
```

Comment:
Depending on whether we really need the `XmlIDREF` annotation, this line might go away.